.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+os: linux
+
+python:
+  - "3.8"
```

Comment:
We said we would request 3.7+ compatibility for public Ensembl repos, shouldn't we test in 3.7 then?